datatypes/src/main/java/com/fasterxml/jackson/datatype/jdk8/WrappedIOException.java

```diff
@@ -0,0 +1,31 @@
+package com.fasterxml.jackson.datatype.jdk8;
+
+import java.io.IOException;
+
+/**
+ * {@link IOException} runtime wrapper
+ * <p>
+ * Wrap an {@link IOException} to a {@link RuntimeException}
+ * </p>
+ */
+public class WrappedIOException extends RuntimeException {
+
+    /**
+     * Constructor
+     *
+     * @param cause IOException to wrap
+     */
+    public WrappedIOException(IOException cause) {
+        super(cause);
+    }
+
+    /**
+     * Returns the wrapped {@link IOException}
+     *
+     * @return the wrapped {@link IOException}
+     */
+    @Override
```

Comment:
You need to close those streams in a finally.
Otherwise if they can hold resources.